Subject: Fix references to scandeps
 Fixed references to scandeps.pl so they refer to scandeps.  The name of the
 program was changed to follow Debian policy.
Last-Update: 2011-04-23
From: Gunnar Wolf <gwolf@debian.org>
Origin: vendor
Forwarded: not-needed
Reviewed-by: Nicholas Bamber <nicholas@periapt.co.uk>
--- a/lib/Module/ScanDeps.pm
+++ b/lib/Module/ScanDeps.pm
@@ -36,12 +36,12 @@
 
 =head1 SYNOPSIS
 
-Via the command-line program L<scandeps.pl>:
+Via the command-line program L<scandeps>:
 
-    % scandeps.pl *.pm          # Print PREREQ_PM section for *.pm
-    % scandeps.pl -e "use utf8" # Read script from command line
-    % scandeps.pl -B *.pm       # Include core modules
-    % scandeps.pl -V *.pm       # Show autoload/shared/data files
+    % scandeps *.pm          # Print PREREQ_PM section for *.pm
+    % scandeps -e "use utf8" # Read script from command line
+    % scandeps -B *.pm       # Include core modules
+    % scandeps -V *.pm       # Show autoload/shared/data files
 
 Used in a program;
 
@@ -1414,7 +1414,7 @@
 
 =head1 SEE ALSO
 
-L<scandeps.pl> is a bundled utility that writes C<PREREQ_PM> section
+L<scandeps> is a bundled utility that writes C<PREREQ_PM> section
 for a number of files.
 
 An application of B<Module::ScanDeps> is to generate executables from
--- a/script/scandeps.pl
+++ b/script/scandeps.pl
@@ -137,20 +137,20 @@
 
 =head1 NAME
 
-scandeps.pl - Scan file prerequisites
+scandeps - Scan file prerequisites
 
 =head1 SYNOPSIS
 
-    % scandeps.pl *.pm          # Print PREREQ_PM section for *.pm
-    % scandeps.pl -e 'STRING'	# Scan an one-liner
-    % scandeps.pl -B *.pm       # Include core modules
-    % scandeps.pl -V *.pm       # Show autoload/shared/data files
-    % scandeps.pl -R *.pm       # Don't recurse
-    % scandeps.pl -C CACHEFILE  # use CACHEFILE to cache dependencies
+    % scandeps *.pm          # Print PREREQ_PM section for *.pm
+    % scandeps -e 'STRING'	# Scan an one-liner
+    % scandeps -B *.pm       # Include core modules
+    % scandeps -V *.pm       # Show autoload/shared/data files
+    % scandeps -R *.pm       # Don't recurse
+    % scandeps -C CACHEFILE  # use CACHEFILE to cache dependencies
 
 =head1 DESCRIPTION
 
-F<scandeps.pl> is a simple-minded utility that prints out the
+F<scandeps> is a simple-minded utility that prints out the
 C<PREREQ_PM> section needed by modules.
 
 If you have B<CPANPLUS> installed, modules that are part of an
